Watchman.AwsResources/Services/Sqs/QueueServiceData.cs

```diff
@@ -4,11 +4,13 @@
 
 namespace Watchman.AwsResources.Services.Sqs
 {
-    public class QueueServiceData
+
+    public class QueueDataV2
     {
-        public QueueData Queue { get; set; }
+        public string Name { get; set; }
 
+        public QueueDataV2 ErrorQueue { get; set; }
```

Comment:
So the model allows an error queue to have an error queue, which probably would never happen (although AWS would support that). This simplifies a few other things though as it now matches other service types, so I think it's ok.